### DISCUSSION TITLE
Missing lake labels

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/mapsforge-creator#35] [Q&A] [answered]
Hi,

Lake labels seem to missing with (almost?) all lakes in Mapsforge maps. Is there a way to get the labels?

**Comment by devemux86** [comment, marked as answer]
The water labels have always been available.
(as names are always included)

![Cruiser](https://user-images.githubusercontent.com/3484020/121799896-81ed1d00-cc37-11eb-80d1-9ecc25ad603f.png)
![Cruiser GL](https://user-images.githubusercontent.com/3484020/121799897-83b6e080-cc37-11eb-8763-724022bfaf2b.png)

**Reply by demiantres** [reply]
My mistake. Thanks for your quick response!
